test/harmonytest.js

```diff
@@ -18524,9 +18524,9 @@ var harmonyTestFixture = {
         },
 
         'func(...a, b)': {
-            index: 9,
+            index: 10,
             lineNumber: 1,
-            column: 10,
+            column: 11,
```

Comment:
This exception is throw on line number 3598 (new), which previously would be invoked before without matching the following comma. However, with this change, the comma *is* matched, which as I understand it, means that the offsets should have increased by 1. And that is the change I made. I agree that it's not the best character to point to, but it basically means that the next token should have been a `)`. Note that had the test been `func(...a , b)`, we'd be facing the same issue, even without my updates.

I'm not sure how to fix it though. Ideas?